static/src/stylesheets/module/external/_from-content-api.scss

```diff
@@ -116,6 +116,11 @@
 .from-content-api pre {
```

Comment:
Not sure I follow. Just ` .from-content-api pre`. pre elements are for pre-formatted text. Code elements are the only usage of those that I know of.